### DISCUSSION TITLE
Please contribute translations for upcoming 3.5 release

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#3694] [Announcements]
Release 3.5 is upcoming.

Please contribute a translation for your language at https://www.apptranslator.org/app/SumatraPDF

Translations with more than 100 untranslated strings will not be included and currently it's over 30 languages.

Namely:
[Burmese](https://www.apptranslator.org/app/SumatraPDF/mm) (320 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=mm))
[Shona](https://www.apptranslator.org/app/SumatraPDF/sn) (302 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=sn))
[Afrikaans](https://www.apptranslator.org/app/SumatraPDF/af) (300 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=af))
[Punjabi](https://www.apptranslator.org/app/SumatraPDF/pa) (279 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=pa))
[Javanese](https://www.apptranslator.org/app/SumatraPDF/jv) (265 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=jv))
[Frisian](https://www.apptranslator.org/app/SumatraPDF/fy-nl) (251 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=fy-nl))
[Malayalam](https://www.apptranslator.org/app/SumatraPDF/ml) (219 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=ml))
[Bengali](https://www.apptranslator.org/app/SumatraPDF/bn) (180 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=bn))
[Uzbek](https://www.apptranslator.org/app/SumatraPDF/uz) (132 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=uz))
[Bosnian](https://www.apptranslator.org/app/SumatraPDF/bs) (131 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=bs))
[Malaysian](https://www.apptranslator.org/app/SumatraPDF/my) (130 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=my))
[Albanian](https://www.apptranslator.org/app/SumatraPDF/sq) (128 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=sq))
[Norwegian Neo-Norwegian](https://www.apptranslator.org/app/SumatraPDF/nn) (128 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=nn))
[Persian](https://www.apptranslator.org/app/SumatraPDF/fa) (128 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=fa))
[Serbian Latin](https://www.apptranslator.org/app/SumatraPDF/sp-rs) (128 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=sp-rs))
[Sinhala](https://www.apptranslator.org/app/SumatraPDF/si) (128 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=si))
[Swedish](https://www.apptranslator.org/app/SumatraPDF/sv) (128 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=sv))
[Kurdish](https://www.apptranslator.org/app/SumatraPDF/ku) (125 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=ku))
[Czech](https://www.apptranslator.org/app/SumatraPDF/cz) (120 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=cz))
[Nepali](https://www.apptranslator.org/app/SumatraPDF/ne) (115 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=ne))
[Macedonian](https://www.apptranslator.org/app/SumatraPDF/mk) (111 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=mk))
[Georgian](https://www.apptranslator.org/app/SumatraPDF/ka) (106 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=ka))
[Irish](https://www.apptranslator.org/app/SumatraPDF/ga) (106 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=ga))
[Slovak](https://www.apptranslator.org/app/SumatraPDF/sk) (106 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=sk))

Other untranslated:
[Hindi](https://www.apptranslator.org/app/SumatraPDF/hi) (94 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=hi))
[Tagalog](https://www.apptranslator.org/app/SumatraPDF/tl) (77 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=tl))
[Latvian](https://www.apptranslator.org/app/SumatraPDF/lv) (75 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=lv))
[Lithuanian](https://www.apptranslator.org/app/SumatraPDF/lt) (73 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=lt))
[Catalan-Valencian](https://www.apptranslator.org/app/SumatraPDF/ca-xv) (67 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=ca-xv))
[Thai](https://www.apptranslator.org/app/SumatraPDF/th) (67 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=th))
[Arabic](https://www.apptranslator.org/app/SumatraPDF/ar) (62 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=ar))
[Azerbaijani](https://www.apptranslator.org/app/SumatraPDF/az) (50 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=az))
[Cornish](https://www.apptranslator.org/app/SumatraPDF/kw) (49 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=kw))
[Armenian](https://www.apptranslator.org/app/SumatraPDF/am) (47 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=am))
[Estonian](https://www.apptranslator.org/app/SumatraPDF/et) (44 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=et))
[Danish](https://www.apptranslator.org/app/SumatraPDF/dk) (14 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=dk))
[Vietnamese](https://www.apptranslator.org/app/SumatraPDF/vn) (13 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=vn))
[Japanese](https://www.apptranslator.org/app/SumatraPDF/ja) (9 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=ja))
[Basque](https://www.apptranslator.org/app/SumatraPDF/eu) (7 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=eu))
[Bulgarian](https://www.apptranslator.org/app/SumatraPDF/bg) (7 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=bg))
[Croatian](https://www.apptranslator.org/app/SumatraPDF/hr) (7 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=hr))
[Hungarian](https://www.apptranslator.org/app/SumatraPDF/hu) (7 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=hu))
[Korean](https://www.apptranslator.org/app/SumatraPDF/kr) (7 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=kr))
[Slovenian](https://www.apptranslator.org/app/SumatraPDF/sl) (7 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=sl))
[Tamil](https://www.apptranslator.org/app/SumatraPDF/ta) (7 untranslated, [rss](https://www.apptranslator.org/rss?app=SumatraPDF&lang=ta))

**Comment by username116** [comment]
I think some translation strings are missing, or I haven't found them, in the right-click menu on a tab: `Properties...` and `Open In New Window` .

**Reply by kjk** [reply]
Good catch thanks. They were not marked for translation, I've fixed it.

**Comment by kyleskimo** [comment]
When creating or editing an annotation, there is a button labeled '**Delete Annotation**', and I think that this string is missing.

**Reply by kjk** [reply]
Good catch, thanks. I've updated the translations.
